packages/lib/getEventTypeById.ts

```diff
@@ -363,18 +362,21 @@ export default async function getEventTypeById({
     bookingFields: getBookingFieldsWithSystemFields(eventType),
   });
 
+  const isOrgEventType = !!eventTypeObject.team?.parentId;
   const teamMembers = eventTypeObject.team
-    ? eventTypeObject.team.members.map((member) => {
-        const user: typeof member.user & { avatar: string } = {
-          ...member.user,
-          avatar: `${CAL_URL}/${member.user.username}/avatar.png`,
-        };
-        return {
-          ...user,
-          eventTypes: user.eventTypes.map((evTy) => evTy.slug),
-          membership: member.role,
-        };
-      })
+    ? eventTypeObject.team.members
+        .filter((member) => member.accepted || isOrgEventType)
```

Comment:
Shouldn't it be?
```suggestion
        .filter((member) => member.accepted || !isOrgEventType)
```

Reply:
Not really, we need to filter out (return false) only when `Accepted = F` and `isOrgEventType = F`. With your alternative we would be filtering out members when `Accepted = F` and `!isOrgEventType = F => isOrgEventType = T` and we want that case to return the member.

| P = Accepted    | Q = isOrgEventType | P ∨ Q |  !Q | P ∨ !Q |
| -------- | ------- | ------- | ------- | ------- |
| T | T | T | F | T |
| T | F | T | T | T |
| F | T | T | F | F |
| F | F | F | T | T |